dist-persist/wbstack/src/Internal/PreApiWbStackUpdate.php

```diff
@@ -16,9 +16,44 @@ public function execute() {
 		@ini_set( 'memory_limit', '-1' ); // also try to disable the memory limit? Is this even a good idea?
 
 		// Run update.php
+		$domain = $GLOBALS[WBSTACK_INFO_GLOBAL]->requestDomain;
 		$mwPath = realpath( __DIR__ . '/../../../' );
-		$cmd = 'WBS_DOMAIN=' . $GLOBALS[WBSTACK_INFO_GLOBAL]->requestDomain . ' php ' . $mwPath . '/maintenance/update.php --quick';
-		exec($cmd, $out, $return);
+		$cmd = 'WBS_DOMAIN=' . $domain . ' php ' . $mwPath . '/maintenance/update.php --quick';
```

Comment:
I think it might be worth trying if we can have the command itself redirect stderr to stdout like this:

```
$cmd = 'WBS_DOMAIN=' . $domain . ' php ' . $mwPath . '/maintenance/update.php --quick 2>&1';
```

which in case it does work could save us a few hoops to jump through in here.